src/main/java/uk/gov/hmcts/reform/coh/domain/SessionEvent.java

```diff
@@ -18,7 +18,7 @@
             @JoinColumn(name = "JURS_ID", referencedColumnName = "jurisdiction_id") })
     private SessionEventForwardingRegister sessionEventForwardingRegister;
 
-    @ManyToOne(fetch=FetchType.LAZY)
+    @ManyToOne(fetch=FetchType.EAGER)
```

Comment:
JPA literally will not pull out the OH not matter how I retrieve it from the SessionEvent. I just gave up and made it eager fetch